packages/api-kit/tests/e2e/addSafeDelegate.test.ts

```diff
@@ -125,10 +126,11 @@ describe('addSafeDelegate', () => {
 
   it('should fail if the signer is not an owner of the Safe', async () => {
     const { safeApiKit, signer } = await getServiceClient(
-      '0xb0057716d5917badaf911b193b12b910811c1497b5bada8d7711f758981c3773'
+      '0xb0057716d5917badaf911b193b12b910811c1497b5bada8d7711f758981c3773',
+      'https://safe-transaction-sepolia.staging.5afe.dev/api'
```

Comment:
This should be removed before merging right? We will use the prod endpoint for the final version

Reply:
Great question. From what I understood, we will merge and then update.

Reply:
@dasanra ?

Reply:
We'll keep it for now